### PR TITLE
Matrix Federation 

### PR DESCRIPTION
**Pull request by expedited**
## Changes 
* Added JSON response to be served in puppet file
* Updated Apache configuration to specify path for puppet file and created an alias for this path. 

Addresses https://github.com/ocf/matrix/issues/1

### CODE REVIEW

File: modules/ocf_www/manifests/site/www.pp [hunk 1]
```diff
@@ -56,6 +69,8 @@
           '%{REQUEST_URI} !^/icons/',
           # ...hide ocfweb metrics
           '%{REQUEST_URI} !^/metrics',
+    # ...and not if it's the matrix well-known file
+    '%{REQUEST_URI} !^/\.well-known/matrix',
```
Comment (nikhiljha) [inline modules/ocf_www/manifests/site/www.pp:73]: The indentation looked right when you were in vim but now it doesn't? Estoy confundido.

File: modules/ocf_www/manifests/site/www.pp [hunk 2]
```diff
@@ -42,6 +49,12 @@
     request_headers     => ['set X-Forwarded-Proto https'],
     proxy_preserve_host => true,
 
+    aliases             => [
+      { alias => '/.well-known/matrix/server',
```
Comment (cg505) [inline modules/ocf_www/manifests/site/www.pp:53]: should probably put alias on its own line to be consistent with the other sections

File: modules/ocf_www/manifests/site/www.pp (hunk 1 above)
Comment (cg505) [reply modules/ocf_www/manifests/site/www.pp:73]: ^^

Comment (expedited) [reply modules/ocf_www/manifests/site/www.pp:73]: I think the linter did that? I wonder if I try to fix it if it will do it again

**Comment by expedited** [comment]
Thanks for handling @cg505 :)

**Comment by cg505** [comment]
it didn't work tho :(

**Pull request merged**